Userland/Services/Taskbar/TaskbarWindow.cpp

```diff
@@ -388,3 +396,37 @@ void TaskbarWindow::set_start_button_font(Gfx::Font const& font)
     m_start_button->set_font(font);
     m_start_button->set_fixed_size(font.width(m_start_button->text()) + 30, 21);
 }
+
+void TaskbarWindow::show_task_button_tooltips()
+{
+    size_t index = 0;
+
+    for_each_visible_taskbar_button([&](auto& button) {
+        if (index >= 10)
+            return IterationDecision::Break;
+
+        if (index >= m_task_button_tooltips.size())
+            m_task_button_tooltips.append(GUI::TooltipWindow::construct());
+
+        VERIFY(index < m_task_button_tooltips.size());
+        auto& window = m_task_button_tooltips[index++];
+
+        auto key = index != 10 ? index : 0;
+        window.set_tooltip(String::formatted("{}", key));
+
+        auto rect = button.screen_relative_rect();
+        Gfx::IntPoint position(rect.center().x(), rect.y() - window.height());
```

Comment:
They look slightly misaligned in the screenshot, I suppose `rect.center().x() - (window.width() / 2)` would solve that?